### PR TITLE
fix(profiles): remove excess mebx random password warning 

### PR DESCRIPTION
**Pull request by intelkrishi**
## PR Checklist

- [ ] Unit Tests have been added for new changes
- [ ] API tests have been updated if applicable
- [ ] All commented code has been removed
- [ ] If you've added a dependency, you've ensured license is compatible with Apache 2.0 and clearly outlined the added dependency.

### CODE REVIEW

File: src/app/profiles/profile-detail/profile-detail.component.ts [hunk 1]
```diff
@@ -119,9 +119,12 @@ export class ProfileDetailComponent implements OnInit {
       this.profileForm.controls.mebxPassword.disable()
       this.profileForm.controls.mebxPassword.setValue(null)
       this.profileForm.controls.mebxPassword.clearValidators()
+      this.profileForm.controls.generateRandomMEBxPassword.setValue(1)
```
Comment (rsdmike) [inline src/app/profiles/profile-detail/profile-detail.component.ts:122]: why `1` ?

File: src/app/profiles/profile-detail/profile-detail.component.ts [hunk 2]
```diff
@@ -356,11 +359,11 @@ export class ProfileDetailComponent implements OnInit {
       const result: any = Object.assign({}, this.profileForm.getRawValue())
       // When creating new
       if ((result.connectionMode === 'CIRA' && result.dhcpEnabled === false) &&
-      (!this.isEdit && (result.generateRandomPassword || result.generateRandomMEBxPassword))) {
+      (!this.isEdit && ((result.activation === Constants.ACMActivate && (result.generateRandomPassword || result.generateRandomMEBxPassword)) || (result.activation === Constants.CCMActivate && result.generateRandomPassword)))) {
```
Comment (rsdmike) [inline src/app/profiles/profile-detail/profile-detail.component.ts:362]: I think this works, however, this whole function has grown to be a bit hard to follow. You wanna take a stab at refactoring a bit? First thing I would look at it reducing the calls to the various dialog functions? And try to reduce the places we call `.onSubmit()?`

File: cypress/integration/profile/create.spec.ts [hunk 3]
```diff
@@ -85,4 +85,42 @@ describe('Test Profile Page', () => {
...
+        .should('eq', httpCodes.CREATED)
+
+      // Check that the config was successful
+      cy.get('mat-cell').contains(profileFixtures.happyPath.profileName)
+      cy.get('mat-cell').contains(profileFixtures.check.network.dhcp.toString())
+      cy.get('mat-cell').contains(profileFixtures.happyPath.ciraConfig)
+      cy.get('mat-cell').contains(profileFixtures.check.mode.ccm)
+    })
+
+    // TODO: check the response to make sure that it is correct
```
Comment (rsdmike) [inline cypress/integration/profile/create.spec.ts:114]: looks like an outstanding todo

**Review by rsdmike — CHANGES_REQUESTED** [review]
I think it looks good, few outstanding todos, and would be awesome to refactor the `confirm()` a bit.

File: src/app/profiles/profile-detail/profile-detail.component.ts (hunk 1 above)
Comment (intelkrishi) [reply src/app/profiles/profile-detail/profile-detail.component.ts:122]: Essentially, I'm setting the checkbox for the MEBX password to yes before disabling it. The 1 is for checking the checkbox.
